### PR TITLE
openPMD_api: New upstream version

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

File: O/openPMD_api/build_tarballs.jl [hunk 1]
```diff
@@ -6,7 +6,7 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "openPMD_api"
-version = v"0.15.1"
+version = v"0.15.2"
 openpmi_api_version = "v.0.14.5" # This is really the `dev` branch after version 0.14.5
 
 # `v"1.6.3"` fails to build
```
Comment (fingolfin) [inline O/openPMD_api/build_tarballs.jl:12]: You may wish to 1.10 while at it (we just added this to libjulia_jll).

**Comment by eschnett** [comment]
I'm currently stuck on [this](https://github.com/JuliaPackaging/Yggdrasil/issues/5851) issue. Apparently, Julia uses an old libstdc++ on some architectures. I will need to see whether this can be updated. (I don't think there will be much progress this week during SC22.)

File: O/openPMD_api/build_tarballs.jl (hunk 1 above)
Comment (eschnett) [reply O/openPMD_api/build_tarballs.jl:12]: @fingolfin `openPMD_api_jll` uses libcxxwrap; does this support Julia 1.10 already?

Comment (fingolfin) [reply O/openPMD_api/build_tarballs.jl:12]: Not yet, waiting for PR #5867 to be merged

Comment (fingolfin) [reply O/openPMD_api/build_tarballs.jl:12]: Never mind, I made #5883 which can be merged now, and it "commutes" with this PR here

Comment (eschnett) [reply O/openPMD_api/build_tarballs.jl:12]: This was now done in a separate PR.

**Comment by eschnett** [comment]
Superseded by https://github.com/JuliaPackaging/Yggdrasil/pull/5902.

**Pull request closed without merging**